test/functional/rpc_getorphantxs.py

```diff
@@ -125,6 +127,29 @@ def orphan_details_match(self, orphan, tx, verbosity):
             self.log.info("Check the transaction hex of orphan")
             assert_equal(orphan["hex"], tx["hex"])
 
+    def test_max_orphan_amount(self):
+        self.log.info("Check that once we reach the max orphan amount we dont get any new orphans")
+        node = self.nodes[0]
+
+        peer_1 = node.add_p2p_connection(P2PInterface())
+
+        self.log.info("Filling up orphanage with " + str(MAX_ORPHANS) + "(MAX_ORPHANS) orphans")
+        for i in range(MAX_ORPHANS):
+            tx_parent_1 = self.wallet.create_self_transfer()
+            tx_child_1 = self.wallet.create_self_transfer(utxo_to_spend=tx_parent_1["new_utxo"])
+            peer_1.send_and_ping(msg_tx(tx_child_1["tx"]))
+
+            orphanage = node.getorphantxs(verbosity=2)
+            assert tx_in_orphanage(node, tx_child_1["tx"])
+        orphanage = node.getorphantxs(verbosity=2)
+        assert_equal(len(orphanage), MAX_ORPHANS)
+
+        self.log.info("Check that we do not add more than the max orphan amount and that the first orphan has been dropped")
+        tx_parent_1 = self.wallet.create_self_transfer()
+        tx_child_1 = self.wallet.create_self_transfer(utxo_to_spend=tx_parent_1["new_utxo"])
+        peer_1.send_and_ping(msg_tx(tx_child_1["tx"]))
+        orphanage = node.getorphantxs(verbosity=2)
```

Comment:
nit: same as above, verbosity shouldn't be needed

Reply:
Thanks! updated in [4d59b9a](https://github.com/bitcoin/bitcoin/pull/31040/commits/4d59b9a517afb5eb661b7bc179c51141769658a3)